specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/sensitivityLabels.json

```diff
@@ -122,13 +129,13 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/sensitivityLabels": {
-      "get": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/schemas/{schemaName}/tables/{tableName}/columns/{columnName}/sensitivityLabels/recommended/enable": {
+      "post": {
         "tags": [
           "SensitivityLabels"
         ],
-        "description": "Gets the sensitivity labels of a given database",
-        "operationId": "SensitivityLabels_ListByDatabase",
+        "description": "Enables sensitivity recommendations on a given column",
```

Comment:
Please add at the end also "(recommendations are enabled by default on all columns)", so it will be clear that there is no need to call this API to enable recommendations

Reply:
Done